test/integration/fake_upstream.cc

```diff
@@ -448,18 +448,17 @@ FakeUpstream::FakeUpstream(const std::string& uds_path, const FakeUpstreamConfig
     : FakeUpstream(Network::Test::createRawBufferSocketFactory(),
                    Network::SocketPtr{new Network::UdsListenSocket(
                        std::make_shared<Network::Address::PipeInstance>(uds_path))},
-                   config) {
-  ENVOY_LOG(info, "starting fake server on unix domain socket {}", uds_path);
-}
+                   config) {}
```

Comment:
The coverage run seems to have crashed around this line.  Could you take a look?

fastbuild/bin/test/integration/uds_integration_test.runfiles/envoy/test/integration/uds_integration_test --gmock_default_mock_behavior=2 '-l trace'
2021-05-12T20:59:56.9789337Z Turning perftools heap leak checking off
2021-05-12T20:59:56.9790321Z [==========] Running 28 tests from 3 test suites.
2021-05-12T20:59:56.9791307Z [----------] Global test environment set-up.
2021-05-12T20:59:56.9792257Z [----------] 10 tests from TestParameters/UdsUpstreamIntegrationTest
2021-05-12T20:59:56.9793085Z [ RUN      ] TestParameters/UdsUpstreamIntegrationTest.RouterRequestAndResponseWithBodyNoBuffer/0
2021-05-12T20:59:56.9794618Z [2021-05-12 20:59:56.357][5547][debug][runtime] [source/common/runtime/runtime_features.cc:31] Unable to use runtime singleton for feature envoy.http.headermap.lazy_map_min_size
2021-05-12T20:59:56.9796679Z [2021-05-12 20:59:56.358][5547][debug][runtime] [source/common/runtime/runtime_features.cc:20] Unable to use runtime singleton for feature envoy.reloadable_features.header_map_correctly_coalesce_cookies
2021-05-12T20:59:56.9798389Z [2021-05-12 20:59:56.362][5547][debug][runtime] [source/common/runtime/runtime_features.cc:31] Unable to use runtime singleton for feature envoy.http.headermap.lazy_map_min_size
2021-05-12T20:59:56.9800154Z [2021-05-12 20:59:56.362][5547][debug][runtime] [source/common/runtime/runtime_features.cc:20] Unable to use runtime singleton for feature envoy.reloadable_features.header_map_correctly_coalesce_cookies
2021-05-12T20:59:56.9801933Z [2021-05-12 20:59:56.363][5547][critical][backtrace] [bazel-out/k8-fastbuild/bin/source/server/_virtual_includes/backtrace_lib/server/backtrace.h:104] Caught Segmentation fault, suspect faulting address 0x0
2021-05-12T20:59:56.9802970Z #0 Envoy::SignalAction::sigHandler() [0x7fcae2c35c94]
2021-05-12T20:59:56.9803623Z #1 __restore_rt [0x7fcad1b20980]
2021-05-12T20:59:56.9804227Z #2 Envoy::FakeUpstream::FakeUpstream() [0x7fcb3f7e953e]
2021-05-12T20:59:56.9804849Z #3 [0x46eab6]
2021-05-12T20:59:56.9805354Z #4 [0x41c56e]
2021-05-12T20:59:56.9805902Z #5 Envoy::BaseIntegrationTest::initialize() [0x7fcb400ebd0e]
2021-05-12T20:59:56.9806574Z #6 Envoy::HttpIntegrationTest::initialize() [0x7fcb406260f5]
2021-05-12T20:59:56.9848246Z #7 Envoy::HttpIntegrationTest::testRouterRequestAndResponseWithBody() [0x7fcb40629d0a]
2021-05-12T20:59:56.9849230Z #8 [0x41180c]
2021-05-12T20:59:56.9849858Z #9 testing::internal::HandleSehExceptionsInMethodIfSupported<>() [0x7fcad299b146]
2021-05-12T20:59:56.9850585Z #10 testing::internal::HandleExceptionsInMethodIfSupported<>() [0x7fcad297fb01]
2021-05-12T20:59:56.9851228Z #11 testing::Test::Run() [0x7fcad2967442]
2021-05-12T20:59:56.9851778Z #12 testing::TestInfo::Run() [0x7fcad2967f58]
2021-05-12T20:59:56.9852278Z #13 testing::TestSuite::Run() [0x7fcad2968839]
2021-05-12T20:59:56.9852852Z #14 testing::internal::UnitTestImpl::RunAllTests() [0x7fcad29765e6]
2021-05-12T20:59:56.9853518Z #15 testing::internal::HandleSehExceptionsInMethodIfSupported<>() [0x7fcad29a2a56]
2021-05-12T20:59:56.9854200Z #16 testing::internal::HandleExceptionsInMethodIfSupported<>() [0x7fcad2982c51]
2021-05-12T20:59:56.9854804Z #17 testing::UnitTest::Run() [0x7fcad2976030]
2021-05-12T20:59:56.9855281Z #18 RUN_ALL_TESTS() [0x7fcaefb86885]
2021-05-12T20:59:56.9855782Z #19 Envoy::TestRunner::RunTests() [0x7fcaefb85abf]
2021-05-12T20:59:56.9856380Z #20 main [0x7fcaefb19b47]
2021-05-12T20:59:56.9857117Z #21 __libc_start_main [0x7fcad173ebf7]
2021-05-12T20:59:56.9857898Z external/bazel_tools/tools/test/collect_coverage.sh: line 131:  5547 Segmentation fault      (core dumped) "$@"